examples/pxScene2d/src/mkapp.sh

```diff
@@ -60,17 +62,16 @@ cp package.json $bundleRes
 if [ "$TRAVIS_EVENT_TYPE" == "cron" ]  
 then
 echo "************ building edge"
-cp pxscene $bundleBin/pxsceneEdge
+cp ${appName} $bundleBin/SparkEdge
 else
-cp pxscene $bundleBin
+cp ${appName} $bundleBin
 fi
 
 if [ "$TRAVIS_EVENT_TYPE" == "cron" ]  
 then
-  sed -i -e 's/\.\/pxscene/\.\/pxsceneEdge/g' macstuff/pxscene.sh
+sed -i -e 's/\.\/Spark/\.\/SparkEdge/g' macstuff/spark.sh
```

Comment:
sed -i -e 's/pxscene.log /pxsceneEdge.log /g' macstuff/pxscene.sh <= this line needs to be added and the names needs to be updated

Reply:
Addressed, please check